distribution/docs/pom.xml

```diff
@@ -265,8 +265,15 @@
                             <goal>copy-resources</goal>
                         </goals>
                         <configuration>
-                            <outputDirectory>${project.build.directory}/doc-contents/images</outputDirectory>
+                            <outputDirectory>${project.build.directory}/images</outputDirectory>
                             <resources>
+                                <resource>
+                                    <directory>src/main/resources/images</directory>
```

Comment:
Out of curiosity, what does this bit do?

Reply:
it takes all the images used in the docs and copies them to a common directory so the renderer can have one central path for image links